File: JS/Pet.js
function Pet(pettype, id, name) {
    this.pettype = pettype;
    this.id = id;
    this.name = name;
    this.size;
    this.color;
    this.age;
}

Pet.prototype.printDataPet = function () {
    console.log(this.pettype)
    console.log(this.id)
    console.log(this.name)
}

File: Java/Pet.java
public class Pet {
    String species;
    String id;
    String name;
    String size;
    String color;
    String age;

    public Pet(String species){
        this.species = species;

    }

    void printDataPet() {
        System.out.println("Species: " + species);
        System.out.println(" ID: " + id + " Name: " + name);
    }
}
